modules/safereachBidAdapter.md

```diff
@@ -0,0 +1,27 @@
+# Overview
+
+```
+Module Name: Safereach SSP Bidder Adapter
+Module Type: Bidder Adapter
+Maintainer: safereachprebid@ukr.net
+```
+
+# Description
+
+Module that connects to Safereach SSP demand sources
+
+# Test Parameters
+```
+    var adUnits = [{
+                code: 'placementCode',
+                sizes: [[300, 250]],
+                bids: [{
+                        bidder: 'safereach',
+                        params: {
+                            placementId: 0,
+                            traffic: 'banner'
```

Comment:
I'm talking about this param. It looks like a "mediaType". What are the other possible values for this param?

Reply:
@mkendall07 other values are native and video, according to this param rtb server handles request